zeq/hbp/vocabulary.cpp

```diff
@@ -60,6 +64,53 @@ zeq::Event serializeCamera( const std::vector< float >& matrix )
     return event;
 }
 
+::zeq::Event serializeImageRaw( const imageRaw& image )
+{
+    ::zeq::Event event( EVENT_IMAGE_RAW );
+    flatbuffers::FlatBufferBuilder& fbb = event.getFBB();
+    auto imageData = fbb.CreateVector( image.data, image.resX * image.resY * RGBA_NBITS );
```

Comment:
Hi,

thank you for your comments. I ammended the changes so
I am not sure if you have been notified.

Cheers,
Greg